bin/link-changelog

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# link creates a link to changelog for the package's version on https://pub.dev.
+#
+# It assumes that the git tag starts with "v".
+
+package_name="${1:-}"
+tag="${2:-}"
+prefix="${3:-}"
+
+if [ -z "$package_name" ]; then
+    echo "error: missing package name"
+    exit 1
+
```

Comment:
Unnecessary empty line (sorry :stuck_out_tongue:).

Reply:
No need to sorry – I'm also pedantic about such minor thingies :P